contrib/tree-sitter-epics-nice-print.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+
+set -e # stop on error
+set -u # undefined/not initialised variables are errors
+set -o pipefail # stop on error in cmd in pipe
+# set -x # print every commands
+
+# color in echo: https://stackoverflow.com/questions/5947742/how-to-change-the-output-color-of-echo-in-linux
+# Black        0;30     Dark Gray     1;30
+# Red          0;31     Light Red     1;31
+# Green        0;32     Light Green   1;32
+# Brown/Orange 0;33     Yellow        1;33
+# Blue         0;34     Light Blue    1;34
+# Purple       0;35     Light Purple  1;35
+# Cyan         0;36     Light Cyan    1;36
+# Light Gray   0;37     White         1;37
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+BLUE='\033[0;34m'
+YELLOW='\033[1;33m'
+CYAN='\033[0;36m'
+NC='\033[0m'        # No Color
+# printf "I ${RED}love${NC} Stack Overflow\n"
+# echo -e "I ${RED}love${NC} Stack Overflow"
+echo -e "color code:"
+echo -e " - ${RED}error${NC}"
+echo -e " - ${YELLOW}warning${NC}"
+echo -e " - ${CYAN}information${NC}"
+echo -e " - ${GREEN}script command${NC}"
+echo -e " - ${BLUE}user input${NC}"
+
+IFS=$'\n\t'
+params="$(
+    getopt \
+        --options "hf:" \
+        --longoptions help,files: \
+        --name "$0" \
+        -- "$@"
+)"
+eval set -- "$params"
+
+printHelp=0
+files=""
+while true; do
+    case "$1" in
+    -h | --help)
+        printHelp=1
+        shift
+        ;;
+    -f | --files)
+        files="$2"
+        shift 2 # shift 2 times: for the option name and option value
+        ;;
+    *)
+        echo -e "${CYAN}[INFO]${NC} rest of the args: $1"
+        break
+        # exit 1
+        ;;
+    esac
+done
+
+if [[ "$printHelp" == 1 ]]; then
+    echo -e "${CYAN}[INFO]${NC} goal of the script: display errors of a specific type of EPICS files in a user-friendly format."
+    echo -e "${CYAN}[INFO]${NC} usage: $0 [-h|--help] [-f|--files \""'$(find <directory> -not -path <exclude> -type f -name <pattern>)'"\" : files to parse] [args...]"
+    echo -e "${CYAN}[INFO]${NC} <directory> : where to find the files, eg: ${HOME}/tops/ "
+    echo -e "${CYAN}[INFO]${NC} <exclude> : to exclude directory from find(), eg: \"./builds/\" "
+    echo -e "${CYAN}[INFO]${NC} <pattern> : find files with this pattern, eg for EPICS files: "*.cmd" or "*.template" or "*.db" or "*.sub*" or "*.st" or "*.proto" "
```

Comment:
Feels a bit weird to have the `[INFO]` in front of the help text.

I also modified the command line so that it supports paths with a space, and added a small example.

```suggestion
    echo -e "goal of the script: display errors of a specific type of EPICS files in a user-friendly format."
    echo -e ""
    echo -e "usage: $0 [-h|--help] [--] <files to parse...>"
    echo -e ""
    echo -e "with find: find <directory> -type f -name <pattern> -not -path <exclude> -print0 | xargs -0 $0"
    echo -e ""
    echo -e "  <directory> : where to find the files, eg: ${HOME}/tops/ "
    echo -e "  <exclude> : to exclude directory from find(), eg: \"./builds/\" "
    echo -e "  <pattern> : find files with this pattern, eg for EPICS files: "*.cmd" or "*.template" or "*.db" or "*.sub*" or "*.st" or "*.proto" "
    echo -e ""
    echo -e "example: find ../myIoc -type f -name "'"*.db"'" -print0 | xargs -0 $0"
```